README.rst

```diff
@@ -2,3 +2,81 @@ openedx-atlas
 #############
 
 An Open edX CLI tool for moving translation files from openedx-translations.
+
+Overview
+--------
+
+OEP-58 proposes changes to the way the Open edX Organization organizes and maintains
+translation files. Atlas is an Open edX CLI tool that uses git's sparse-checkout to
+download directories with the repository openedx-translations. These directories
+correspond to repositories within the GitHub openedx organization. They contain
+translation files downloaded from Transifex that have been translated by the Open edX
+Translations Working Group.
+
+Atlas is intended for both development and deployment, and is meant to be used after
+cloning a repository with translation files kept in openedx-translations. For instance,
+when building a docker image or testing localization strings locally. It should not be
+necessary to run any application in English.
+
+Usage
+-----
+
+Atlas is a CLI tool that has essentially one command: `atlas pull`
+
+Atlas defaults to using a configuration file named `atlas.yaml` placed
+in the root directory. Configuration file:
+
+pull:
+  branch: <branch-name>
+  directory: <directory-name>
+  repository: <organization-name>/<repository-name>
+
+Atlas can also use a configuration file in a different path using the `--config` flag
+after `atlas`: `atlas --config pull`.
+
+Atlas can also be used without a configuration file by using the flags below after
+`atlas pull`.
+
+`-b` or `--branch`
+`-r` or `--repository`
+`-d` or `--directory`
+
+Documentation
+-------------
+
+TODO
+
+License
+-------
+
+The code in this repository is licensed under the AGPL 3.0 unless otherwise noted.
+
+Please see ``LICENSE`` for details.
+
+How To Contribute
+-----------------
+
+Contributions are very welcome.
+
+Please read
+`How To Contribute <https://github.com/openedx/edx-platform/blob/master/CONTRIBUTING.rst>`_
```

Comment:
You could also link them to https://openedx.atlassian.net/wiki/spaces/COMM/pages/941457737/How+to+start+contributing+to+the+Open+edX+code+base which I believe is more up-to-date.

Reply:
Linked to this page since its more up to date